activesupport/lib/active_support/concern.rb

```diff
@@ -104,6 +104,12 @@ def initialize
       end
     end
 
+    class UnreachableIncludedBlock < StandardError #:nodoc:
+      def initialize
+        super "Cannot add an 'included' block after Concern was already included"
```

Comment:
That's a good idea, I'll adjust the error.

I didn't manage to resolve the issue by explicitly requiring either the class in the concern file, or the concern in the class file. Both times I got the error:

```
Circular dependency detected while autoloading constant [NameOfClass/NameOfConcern] (RuntimeError)
```